src/index/header.scss

```diff
@@ -27,6 +29,15 @@
     background-image: url('../images/logo_validators_dapp.png');
     background-position: 0 0;
   }
+  
+  &-logo-sokol {
+    @include image-2x('../images/logo_sokol@2x.png', 146px, 35px);
+    float: left;
+    width: 190px;
+  	height: 35px;
+    background-image: url('../images/logo_sokol.png');
+    background-position: 0 0;
```

Comment:
Let's add `background-repeat: no-repeat` due to 
![screen shot 2018-01-25 at 17 27 11](https://user-images.githubusercontent.com/4341812/35393107-0cf3ce5a-01f5-11e8-9730-5d19f5c35edd.png)

Reply:
The rest looks good. I vote to merge it after this fix.